evm/package.json

```diff
@@ -14,8 +15,9 @@
     "pretest": "yarn build",
     "test": "truffle test ./dist/test/*",
     "format": "prettier --write \"**/*\"",
-    "truffle:migrate:cldev": "truffle migrate --network cldev",
-    "prepublishOnly": "yarn build && yarn lint && yarn test"
+    "prepublishOnly": "yarn build && yarn lint && yarn test",
+    "setup": "cross-env ./scripts/build.js",
```

Comment:
needless cross-env?

Reply:
It's required here because windows can't understand `/` in the path